charts/iter8/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: iter8
-version: 0.14.3
+version: 0.15.0
```

Comment:
... and this is `v0.15.0`. Is this what we want?

Reply:
we want this one